adapters/flipp/flipp_params.go

```diff
@@ -0,0 +1,315 @@
+package flipp
+
+type Keyword string
+
+type CampaignRequestBodyUser struct {
+
+	// Key to keep track of user for retargeting purposes.
+	// Required: true
+	Key *string `json:"key"`
+}
+
+// Location location
+//
+// swagger:model Location
+type Location struct {
+
+	// accuracy radius
+	AccuracyRadius int64 `json:"accuracy_radius,omitempty"`
+
+	// city
+	City string `json:"city,omitempty"`
+
+	// country
+	Country string `json:"country,omitempty"`
+
+	// ip
+	IP string `json:"ip,omitempty"`
+
+	// metro code
+	MetroCode string `json:"metro_code,omitempty"`
+
+	// postal code
+	PostalCode string `json:"postal_code,omitempty"`
+
+	// region
+	Region string `json:"region,omitempty"`
+
+	// region name
+	RegionName string `json:"region_name,omitempty"`
+}
+
+// Properties properties
+//
+// swagger:model Properties
+type Properties struct {
+
+	// content code
+	ContentCode *string `json:"contentCode,omitempty"`
+
+	// loc
+	Loc Location `json:"loc"`
```

Comment:
We shouldn't be expecting partners will provide this, and should probably omit location to reduce confusion/complexity

We do need the IP, but that's set at the request level

Reply:
removed